xla/mlir/tools/mlir_interpreter/dialects/BUILD

```diff
@@ -61,3 +63,33 @@ cc_library(
         "@llvm-project//mlir:ViewLikeInterface",
     ],
 )
+
+cc_library(
+    name = "symbol_finder_header",
+    hdrs = ["symbol_finder.h"],
+)
```

Comment:
We try not to have these header targets that don't work without an implementation.

Can you do the following instead:

1. Remove this target
2. Add the header file name to all 3(!) cc_library targets below.
3. For `symbol_finder_linux` you add it into `srcs` instead of `hdrs`
4. For `symbol_finder_windows` you do the same - you add it into `srcs` instead of `hdrs`
5. In `find_symbol` you should add just as a normal headers in `hdrs`.

Reply:
Hey, I think this is still missing. If it is unclear what to do , let me know.

Reply:
Sorry, I missed it. Please review the changes made